src/cider-velox/src/CiderHashJoinBuild.cpp

```diff
@@ -22,7 +22,12 @@
 #include "CiderHashJoinBuild.h"
 #include "Allocator.h"
 #include "velox/exec/Task.h"
+
+#ifndef CIDER_BATCH_PROCESSOR_CONTEXT_H
```

Comment:
why need this?

Reply:
I am using cider::exec::nextgen::context::Batch, and in Batch.h there is ```exec/module/batch/ArrowABI.h``` which has redefinition with ```velox/vector/arrow/Abi.h```. They both contains same struct of ArrowSchema, ArrowArray and ArrowArrayStream.